assets/src/components/notificationCard.tsx

```diff
@@ -45,40 +56,55 @@ export const NotificationCard = ({
       noFocusOrHover={noFocusOrHover}
     >
       <CardBody>{description(notification, routes, routeAtCreation)}</CardBody>
-      <CardProperties
-        properties={[
-          {
-            label: "Run",
-            value:
-              notification.runIds.length > 0
-                ? notification.runIds.join(", ")
-                : null,
-          },
-          {
-            label: "Operator",
-            value:
-              notification.operatorName !== null &&
-              notification.operatorId !== null
-                ? `${notification.operatorName} #${notification.operatorId}`
-                : null,
-            sensitive: true,
-          },
-        ]}
-      />
+      {isBlockWaiverNotification(notification) && (
```

Comment:
Why is this new conditional needed when it wasn't before? Does this mean that this section was previously rendered for BridgeMovements, but it shouldn't have been?

Reply:
These properties _could_ render a Bridge Movement notification, if it had runIds and/or an operator name, but in practice, we prevented those states on the backend.

On the backend, a bridge notification would always an empty run_ids key https://github.com/mbta/skate/blob/main/lib/notifications/notification.ex#L162-L172 and operator name and ID is left out entirely. So it would be impossible for the backend to produce a bridge notification which triggered this UI logic.

You will find that due to this lack of type safety, our tests used to expect that a bridge notification would render with run ids, and would look for it when trying to click elements 
https://github.com/mbta/skate/blob/main/assets/tests/components/notificationCard.test.tsx#L250
[which had to be changed](https://github.com/mbta/skate/pull/2773/commits/78c9cf9b27b521e4aacf316eca944222219ec164#diff-54dea08de39a5c7398212550c06dcad3fd2c2f6af9f39e1c0718d0cb869c0038L250-R349)